packages/analytics/e2e/analytics.e2e.js

```diff
@@ -1133,5 +1134,28 @@ describe('analytics() modular', function () {
         await setConsent(getAnalytics(), consentSettings);
       });
     });
+
+    describe('getGoogleAnalyticsClientId()', function () {
+      it('Error when trying to use getGoogleAnalyticsClientId() on non-web platforms', async function () {
+        if (!Platform.other) {
+          try {
+            const { getAnalytics } = analyticsModular;
+            await firebase.analytics.getGoogleAnalyticsClientId(getAnalytics());
+            fail('Should have thrown an error');
+          } catch (e) {
+            e.message.should.equal('getGoogleAnalyticsClientId is web-only.');
+          }
+        }
+        this.skip();
```

Comment:
does this work? Or should it be:

```suggestion
        } else {
          this.skip();
        }
```

🤔  (same below)